vignettes/adrs.Rmd

```diff
@@ -719,4 +745,4 @@ dataset_vignette(
 
 ADaM | Sample Code
 ---- | --------------
-ADRS | [ad_adrs.R](https://github.com/pharmaverse/admiralonco/blob/main/inst/templates/ad_adrs.R){target="_blank"}
+ADRS | `r dyn_link(ad_adrs.R, "https://github.com/pharmaverse/admiralonco/blob", "inst/templates/ad_adrs.R")`
```

Comment:
This fails because it should be `"ad_adrs.R"` instead of `ad_adrs.R`.

And `is_multiversion = TRUE` should be used. Otherwise "main" is not added to the link.

Reply:
thanks - i was just getting around to looking at why these failed. you saved me a job 😃

Reply:
updated now